ktoml-core/src/linuxX64Main/kotlin/com/akuleshov7/ktoml/utils/RegexesLinux.kt

```diff
@@ -0,0 +1,9 @@
+/**
+ * Regexes used in tree writing.
+ */
+
+package com.akuleshov7.ktoml.utils
+
+internal actual val controlCharacterRegex = commonControlCharacterRegex
```

Comment:
actually there is no need to create 3 directories for this logic. You can create a gradle task: `nativeMain`. Here is a good example of this: https://github.com/analysis-dev/save-cli/blob/c4414a8caa023900b62ef2ab4b8628ec35c8510f/save-cli/build.gradle.kts


```
        val nativeMain by creating {
            dependsOn(commonMain)
            dependencies {
                implementation(projects.saveCore)
                implementation(libs.kotlinx.serialization.properties)
            }
        }
        saveTarget.forEach {
            getByName("${it.name}Main").dependsOn(nativeMain)
        }
```

Reply:
we can even have naiveNonJvm that will join JS, linux, mac, win and many new targets